src/main/java/edu/harvard/iq/dataverse/DataversePage.java

```diff
@@ -511,18 +511,6 @@ public void hideDatasetFieldTypes(Long mdbId) {
         }
         setEditInputLevel(false);
     }
-    
-    public void toggleInputLevel( Long mdbId, long dsftId){
```

Comment:
Another question: this is being removed, but I don't see a corresponding removal if it being called? Was it never used (or am I just missing it)?

Reply:
That was what was being called in the (now removed) onclick on the option buttons and nowhere else.